### ISSUE TITLE
Mouse and stylus stop working after trying to open brush selection dialog on the bottom bar.

### ISSUE THREAD

**Issue by lohroc** [mypaint/mypaint#1201]
#### Description of the problem

Mouse and stylus stop working after trying to open brush selection dialog on the bottom bar.

#### Basic system details

MyPaint version: 2.0.1
Operating system: Fedora 38
Desktop environment: GNOME

#### Steps to reproduce

Open the brush selection dialog.

#### Backtraces or error messages

```
(mypaint:42663): Gdk-WARNING **: 22:20:02.846: Window 0x56202541e050 is already mapped at the time of grabbing. gdk_seat_grab() should be used to simultanously grab input and show this popup. You may find oddities ahead.
WARNING: gui.windowing: grab: outside-popup grab already active: regrabbing

(mypaint:42663): Gdk-WARNING **: 22:20:13.568: Window 0x56202541e050 is already mapped at the time of grabbing. gdk_seat_grab() should be used to simultanously grab input and show this popup. You may find oddities ahead.
WARNING: gui.windowing: grab: outside-popup grab already active: regrabbing

(mypaint:42663): Gdk-WARNING **: 22:20:20.005: Window 0x56202541e050 is already mapped at the time of grabbing. gdk_seat_grab() should be used to simultanously grab input and show this popup. You may find oddities ahead.

```

**Comment by asluizh** [comment]
Same issue here. Nothing on the screen responds to clicks, touch or pen. The window will still refresh, and the message will change depending on whether I am holding shift, control or alt.

These are the last few lines from the terminal: 
```
(mypaint:46542): Gdk-WARNING **: 16:23:24.916: Window 0x5565b22a9bf0 is already mapped at the time of grabbing. gdk_seat_grab() should be used to simultanously grab input and show this popup. You may find oddities ahead.
INFO: gui.device: New device 'Wayland Finger Scrolling' (GDK_SOURCE_TOUCHPAD, axes:2, class=GdkWaylandDevice, vendor=None, product=None)
```

Also on Fedora 38, Gnome wayland, MyPaint version 2.0.1+gitexport.478280f.

**Comment by mad4linux** [comment]
Same problem on Debian 12. It is only happening on wayland, works fine with X11. I'm having the issue with the color dialog as well. Both dialogs are displayed in the top left corner of the canvas. Here's my very similar GDK warning:
(mypaint:4841): Gdk-WARNING **: 21:31:47.714: Window 0x3a0b3c0 is already mapped at the time of grabbing. gdk_seat_grab() should be used to simultanously grab input and show this popup. You may find oddities ahead.

**Comment by mad4linux** [comment]
It seems to be a duplicate of #1161. 
What i forgot to mention: Debian 12 has mypaint version 2.0.1+gitexport.478280f.

**Comment by AesaraB** [comment]
Closed as duplicate of #1161
